src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.util.Scanner;
+import command.ListCommand;
+import command.MarkCommand;
+import command.UnmarkCommand;
+import data.task.builder.DeadlineBuilder;
+import data.task.builder.EventBuilder;
+import data.task.builder.TodoBuilder;
+import command.AddTaskCommand;
+import command.DeleteCommand;
+import command.FindCommand;
+import exception.DukeException;
+import ui.Invoker;
+
 public class Duke {
+    private Invoker invoker = new Invoker();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Doctor101");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Duke duke = new Duke();
+        duke.invoker.setCommand("list", new ListCommand());
```

Comment:
I like the abstraction of the `Invoker` class and the `setCommand` function. It allows for easier extension in the future.